src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.Paint.cs

```diff
@@ -175,6 +184,24 @@ public void ClipToContent(ShadowPaintState state, SKCanvas canvas)
 			canvas.ClipRoundRect(GetContentShape(state), antialias: true);
 		}
 
+		public static bool IsCornerRadiusProportional(CornerRadius CornerRadius)
+		{
+			return (float)CornerRadius.TopLeft == (float)CornerRadius.TopRight &&
+						(float)CornerRadius.TopRight == (float)CornerRadius.BottomRight &&
+						(float)CornerRadius.BottomRight == (float)CornerRadius.BottomLeft;
+		}
+
+		public static SKPoint[] GetPointsFromCornerRadius(CornerRadius CornerRadius, float pixelRatio)
+		{
+			return new SKPoint[]
```

Comment:
return new[] { CornerRadius.TopLeft, .TR, .BR, .BL }
	.Select(x => (float)x * pixelRatio))
	.Select(x => new SKPoint(x, x))
	.ToArray();

Reply:
maybe consider inline this into GetContentShape's else branch
since this is not reused, and doesnt really encapsulate much

Reply:
Done.